pom.xml

```diff
@@ -336,6 +356,17 @@
           <artifactId>jacoco-maven-plugin</artifactId>
           <version>0.7.8</version>
         </plugin>
+
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>2.5.1</version>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>2.12.4</version>
```

Comment:
A semi-mistake: it was from the `unit` branch for the in-bundle unit-tests.  The `c.g.c.t.e.appengine.standard.java8` bundle includes some such unit tests and includes these plugins with a version. I'll remove those versions to pick up from here.